powerhome/playbook
# [PLAY-693] Fixing Tooltip kit wrong backgrounds in Rails

#### Screens

![image](https://user-images.githubusercontent.com/2573205/230419564-706a5f16-f945-418d-8e0c-989455393983.png)

![Screenshot 2023-04-06 at 11 58 18](https://user-images.githubusercontent.com/2573205/230419750-230cda5f-32bf-43f9-80e6-8504bab21c57.png)


#### Breaking Changes

No

#### Runway Ticket URL

[Runway Ticket](https://nitro.powerhrg.com/runway/backlog_items/PLAY-693)

#### How to test this

Go to Tooltip docs in Rails and hover over any doc examples
toggle to dark mode to see the erroneous style there, as well

#### Checklist:
